packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -127,6 +127,22 @@ const extraConfig = system({
     property: 'borderColor',
     scale: 'borderColors',
   },
+  borderBottomColor: {
+    property: 'borderBottomColor',
+    scale: 'borderColors',
+  },
+  borderLeftColor: {
+    property: 'borderLeftColor',
+    scale: 'borderColors',
+  },
+  borderRightColor: {
+    property: 'borderRightColor',
+    scale: 'borderColors',
+  },
+  borderTopColor: {
+    property: 'borderTopColor',
+    scale: 'borderColors',
+  },
```

Comment:
These were working from the colors key and we kinda didn't notice

Reply:
Good catch.